frontend/libs/studio-components/src/components/StudioSectionHeader/StudioSectionHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { forwardRef, type HTMLAttributes } from 'react';
+import { Heading, type HeadingProps, HelpText } from '@digdir/design-system-react';
+
+import classes from './StudioSectionHeader.module.css';
+import cn from 'classnames';
+
+export type StudioSectionHeaderProps = {
+  icon?: React.ReactNode;
+  heading: {
+    text: string;
+    level?: HeadingProps['level'];
+  };
+  helpText: {
+    text: string;
+    title: string;
+  };
+} & HTMLAttributes<HTMLDivElement>;
+const StudioSectionHeader = forwardRef<HTMLDivElement, StudioSectionHeaderProps>(
```

Comment:
Med et linjeskift her blir det lettere å se hvor komponentkoden starter:
```suggestion
} & HTMLAttributes<HTMLDivElement>;

const StudioSectionHeader = forwardRef<HTMLDivElement, StudioSectionHeaderProps>(
```

Reply:
Bommert. pleier å gjøre det, men den lille teksten med referanse til hvor komponenten er bruk lurer meg i blant. Hadde ikke den i VSCode. 🙈

Reply:
Hehe skjønner. Jeg har plassert den på siden i stedet for å unngå dette. Settings -> Editor -> Inlay hints -> Code vision -> Default position for metrics.
<img width="483" alt="image" src="https://github.com/Altinn/altinn-studio/assets/29770305/f6ebf887-1751-4f74-917b-a2f0cf316b0a">